cpanspec

```diff
@@ -875,16 +875,19 @@ for my $ofile (@args) {
     foreach my $entry (@archive_files) {
         my $version0=$version;
         $version0=~s/0*$/0*/; # pathnames may not contain as many trailing zeros
-        if ($entry !~ /^(?:.\/)?($name-(?:v\.?)?$version0)(?:\/|$)/) {
+        if (
+            $entry !~ /^(?:.\/)?($name-(?:v\.?)?$version0)(?:\/|$)/
+            and $entry !~ /^(?:.\/)?($name)(?:\/|$)/
```

Comment:
I very much prefer `&&` for `if` conditions.

Reply:
I really prefer `and` and `or` :)